charts/hivemq-platform/values.yaml

```diff
@@ -101,7 +101,24 @@ license:
   # The name of the Kubernetes Secret with all HiveMQ licenses.
   # To create a new Kubernetes Secret with licenses, set create=true. Otherwise, false to reuse an existing Secret.
   name: ""
-  # Inlines the main HiveMQ license as an encoded 64-bytes string.
+  # Defines specific annotations to be applied to the Kubernetes Secret that contains the HiveMQ licenses.
+  annotations: {}
+  # Defines specific labels to be applied to the Kubernetes Secret that contains the HiveMQ licenses.
+  labels: {}
+  # Configures whether to use a Base64 encoded string (data) or clear content (stringData) for the rendered Kubernetes Secret.
+  # This is useful when using some third party tools to handle the license content through placeholders, such as the ArgoCD Vault plugin.
+  # For example, the following Helm command:
+  #   `helm template platform hivemq/hivemq-platform --set license.create=true --set license.isLicenseBase64Encoded=false --set license.data=<hivemq-license-placeholder>`
+  # will render the following Kubernetes Secret:
+  #   apiVersion: v1
+  #   kind: Secret
+  #   metadata:
+  #     name: hivemq-license-platform
+  #   stringData:
+  #     license.lic: <hivemq-license-placeholder>
+  isLicenseBase64Encoded: true
+  # Inlines the main HiveMQ license as a string.
+  # By default, this data must be a Base64 encoded string. Otherwise, set isLicenseBase64Encoded=false to use a clear string.
```

Comment:
By default, this data is expected as Base64 encoded string. Set "isLicenseBase64Encoded=false" to provide clear content.